test/push_neg.lean

```diff
@@ -0,0 +1,28 @@
+import tactic.push_neg
+
+example (h : ∃ p: ℕ, ¬ ∀ n : ℕ, n > p) (h' : ∃ p: ℕ, ¬ ∃ n : ℕ, n < p) : ¬ ∀ n : ℕ, n = 0 :=
```

Comment:
One of the selling points of your tactic is preserving the name of bound variables right? I don't think that you test that particular feature

Reply:
I don't know how to test this.

Reply:
First thing to check: if the naming is wrong, does your current test catch the problem?

Reply:
Of course it doesn't, the assumption with wrong naming is defeq to the correct one. But I don't know how to check for syntactic equality of target or assumption.

Reply:
comparing expressions using `=` will detect renaming

Reply:
Done